src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -78,23 +78,31 @@ <h1 class="subheader__title">
                         {% if course.duration or course.effort or current_page.publisher_is_draft %}
                         <div class="characteristics">
                             <ul class="characteristics__container">
-                            {% render_model_block course "duration,effort" %}
+                            {% render_model_block course "duration,effort,is_self_paced" %}
                                 {% if instance.duration or current_page.publisher_is_draft %}
                                 <li class="characteristics__item">
                                     <svg role="img" class="characteristics__icon" aria-hidden="true">
-                                        <use href="#icon-clock" />
+                                        <use href="#icon-calendar" />
                                     </svg>
                                     <span class="characteristics__term">{% trans "Duration:" %} {{ instance.get_duration_display|default:"NA" }}</span>
                                 </li>
                                 {% endif %}
                                 {% if instance.effort or current_page.publisher_is_draft %}
                                 <li class="characteristics__item">
                                     <svg role="img" class="characteristics__icon" aria-hidden="true">
-                                        <use href="#icon-stopwatch" />
+                                        <use href="#icon-clock" />
                                     </svg>
                                     <span class="characteristics__term">{% trans "Effort:" %} {{ instance.get_effort_display|default:"NA" }}</span>
                                 </li>
                                 {% endif %}
+                                {% if instance.get_pace_display or current_page.publisher_is_draft %}
+                                <li class="characteristics__item">
+                                    <svg role="img" class="characteristics__icon" aria-hidden="true">
+                                        <use href="#icon-stopwatch" />
```

Comment:
maybe this kind of meter icon is more explicit:
![image](https://user-images.githubusercontent.com/1427165/109785247-3bd1f780-7c0c-11eb-95ae-5eeaf28b5da4.png)

Reply:
![Pace Copy](https://user-images.githubusercontent.com/9265241/109800635-b9eaca00-7c1d-11eb-8d0b-32c2e4003be9.png)
I finally use this one. (Take a look to screenshots joined in description)